test/all-path/00-basic/03-concurrent-rules/all-path-b-or-c-spec.k

```diff
@@ -8,7 +8,6 @@ module ALL-PATH-B-OR-C-SPEC
   imports PATH
 
 // This should be provable as both one-path and all-path.
-  rule <k> a => ?X </k> 
-    ensures ?X ==K b orBool ?X ==K c [all-path]
+  rule <k> a => b #Or c </k> [all-path]
```

Comment:
Will this still work with an existential variable instead?

Reply:
Yes, I decided to leave a few tests with the disjunction in the condition and a few with it in the term, to make sure they pass in both cases.